paddle/math/MathFunctions.cpp

```diff
@@ -180,7 +180,6 @@ int getri<double>(const CBLAS_ORDER order,
                   const int lda,
                   const int* ipiv) {
   return dynload::PADDLE_DGETRI(order, N, A, lda, ipiv);
-  return 0;
```

Comment:
I am far from an expert of the `math` package. Just curious if this change is related to Android?